### DISCUSSION TITLE
ToggleSwitch Disabled Opacity

### DISCUSSION THREAD

**Discussion by dvazriel** [MahApps/MahApps.Metro#4344] [Q&A]
Is there a way to take the ToggleSwitch "read-only", just to use it to display status but prevents the user from toggling it.  Disabling changes the opacity of the control - i want to keep it at 100%.

Any ideas?

thanks!

**Comment by timunie** [comment]
Hi @dvazriel 

You may want to override these Brushes: https://github.com/MahApps/MahApps.Metro/blob/8d46a8a8447085887703f0ab0ecae4ad95ec9289/src/MahApps.Metro/Themes/ToggleSwitch.xaml#L388-L400

Happy coding
Tim

**Reply by dvazriel** [reply]
Hi Tim, thanks for the reply.

Where would I write the Brush overrides? in the App.xaml? - sorry I am new to WFP. Could you provide an example? I am guessing i need to change all those` Value="{DynamicResource MahApps.Brushes.ToggleSwitch.ContentForegroundDisabled}"` to something like
`Value="{DynamicResource MahApps.Brushes.ToggleSwitch.ContentForegroundEnabled}"`

Thanks again for the help.

**Reply by timunie** [reply]
Yes, you can override it in App.xaml. Have a look in here how it's done: https://github.com/MahApps/MahApps.Metro/blob/8d46a8a8447085887703f0ab0ecae4ad95ec9289/src/MahApps.Metro/Styles/Themes/Theme.Template.xaml#L526-L558

Note to add the following line: https://github.com/MahApps/MahApps.Metro/blob/8d46a8a8447085887703f0ab0ecae4ad95ec9289/src/MahApps.Metro/Styles/Themes/Theme.Template.xaml#L3
